plugin/code_actions.py

```diff
@@ -178,7 +174,7 @@ def filtering_collector(
     """
 
     def actions_filter(actions: CodeActionsResponse) -> List[CodeActionOrCommand]:
-        return [a for a in (actions or []) if a.get('kind') in kinds]
+        return [a for a in (actions or []) if a.get('kind') in kinds]  # type: ignore
```

Comment:
The `get('kind')` triggers type error as `Command` can't have `kind` attribute.

Even checking with `'kind' in a` doesn't help.